Assets/LeapMotion/Modules/InteractionEngine/Scripts/Editor/InteractionBehaviourEditor.cs

```diff
@@ -32,6 +35,105 @@ public class InteractionBehaviourEditor : CustomEditorBase<InteractionBehaviour>
                                 "_moveObjectWhenGrasped",
                                 "graspedMovementType",
                                 "graspHoldWarpingEnabled__curIgnored");
+
+      // Layer Overrides
+      specifyConditionalDrawing(() => target.overrideInteractionLayer,
+                                "_interactionLayer");
+      specifyConditionalDrawing(() => target.overrideNoContactLayer,
+                                "_noContactLayer");
+      specifyCustomDecorator("_noContactLayer", drawNoContactLayerDecorator);
+    }
+
+    private void drawInteractionManagerDecorator(SerializedProperty property) {
+      if (PrefabUtility.GetPrefabType(target) == PrefabType.Prefab) {
```

Comment:
This editor specifies that it is allowed to edit more than one object at a time, and so you should refrain from using the 'target' keyword, since it will only allow you to make decisions based on the state of a single object.